### ISSUE TITLE
Permissions: /opt/api-umbrella/var/tmp

### ISSUE THREAD

**Issue by srikraj8341** [NREL/api-umbrella#352]
I am trying to install API umbrella on Amazon EC2 instance on Ubuntu 14.04 version from source code. 
While running "sudo make install" command, the below is occuring 

 Replacing: /opt/api-umbrella/embedded/openresty/luajit/bin/luarocks-5.1 /opt/api-umbrella/embedded/openresty/luajit/bin/luarocks-admin-5.1 /opt/api-umbrella/embedded/openresty/luajit/share/lua/5.1/luarocks/site_config.lua
-- Permissions: /opt/api-umbrella/var/tmp

Can some one suggest me what can be done to rectify this error and go forward with smooth installation

**Comment by GUI** [comment]
I'm not sure I see the error here. This looks like the normal output from a successful `sudo make install`. Is something not working as expected after the make install?

**Comment by srikraj8341** [comment]
Yeah I am not able to start api-umbrella. Getting the below error

[....] Starting api-umbrella (via systemctl): api-umbrella.serviceFailed to start api-umbrella.service: Unit api-umbrella.service not found.

**Comment by srikraj8341** [comment]
Or How to start api umbrella in this fashion ? while compiling from source code
